docs/architecture/cloud-native/relational-vs-nosql-data.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Relational vs. NoSQL data
+description: Learn about relational and NoSQL data in cloud-native applications
+author: robvet
+ms.date: 12/30/2019
+---
+# Relational vs. NoSQL data
+
+[!INCLUDE [book-preview](../../../includes/book-preview.md)]
+
+## Introduction
+
+Relational and NoSQL are two types of database systems commonly implemented in cloud-native apps. They're built differently, store data differently, and accessed differently. In this section, we'll look at both. Later in this module, we'll look at an emerging database technology called *NewSQL*.
+
+*Relational databases* have been a prevalent technology for decades. They're mature, proven, and widely implemented. Competing database products, tooling, and expertise abound. Relational databases provide a store of related data tables. These tables have a fixed schema, use SQL (Structured Query Language) to manage data, and support [ACID](https://www.geeksforgeeks.org/acid-properties-in-dbms/) guarantees.
+
+*No-SQL databases* refer to high-performance, non-relational data stores. They excel in their ease-of-use, scalability, resilience, and availability characteristics. Instead of joining tables of normalized data, NoSQL stores unstructured or semi-structured data, often in key-value pairs or JSON documents. No-SQL databases typically don't provide [ACID](https://www.geeksforgeeks.org/acid-properties-in-dbms/) guarantees beyond the scope of a single database partition. High volume services that require sub second response time favor NoSQL datastores.
+
+The impact of [NoSQL](https://www.geeksforgeeks.org/introduction-to-nosql/) technologies for distributed cloud-native systems can't be overstated. The proliferation of new data technologies in this space has disrupted solutions that once exclusively relied on relational databases.
+
+NoSQL databases include several different models for accessing and managing data, each suited to specific use cases. Figure 5-9 presents four common models.
+
+![NoSQL data models](./media/types-of-nosql-datastores.png)
+
+**Figure 5-9**: Data models for NoSQL databases
+
+## The CAP theorem
+
+As a way to understand the differences between these types of databases, consider the [CAP theorem](https://towardsdatascience.com/cap-theorem-and-distributed-database-management-systems-5c2be977950e), a set of principles applied to distributed systems that store state. Figure 5-10 shows the three properties of the CAP theorem.
+
+![CAP theorem](./media/cap-theorem.png)
+
+**Figure 5-10**. The CAP theorem
```

Comment:
IMO, the font used the image is not correct. Can you use a consistent font across all images?

Reply:
Fixed.